### PR TITLE
Feature finetuning comments for pods

### PR DESCRIPTION
**Pull request by ishandutta2007**
Here I have added two features:

1. added different levels for amount of comments desired to receive. (10%,30%,90%)
2. addressed the issue to prevent multiple times commenting on same post.

The featues discussed [here](https://github.com/timgrossmann/InstaPy/pull/4308#issuecomment-483522098) will be done after merging these.

There has been serverside changes made including db fields. But it is expected to be fully backward compatible.

@timgrossmann [Here](https://github.com/InstaPy/instapy-pods-server/pull/2) are the corresponding server side changes.

### CODE REVIEW

**Comment by timgrossmann** [comment]
@ishandutta2007 Sorry for the incredibly long delay... Any chance you could rebase the PR to the current dev branch? 🙈

**Comment by ishandutta2007** [comment]
> @ishandutta2007 Sorry for the incredibly long delay... Any chance you could rebase the PR to the current dev branch? 🙈

merging shouldn't be an issue for you as I had branched out after all other PRs currently here. Let us know if you get any merge conflict you can't resolve.

File: instapy/instapy.py [hunk 1]
```diff
@@ -5322,37 +5332,74 @@ def join_pods(self, topic='general'):
                 self.logger.info("I don't have any recent post, so I will just help a few pod posts and move on.")
                 nposts = 40
 
-            if len(pod_post_ids) > nposts:
-                pod_post_ids = random.sample(pod_post_ids, nposts)
+            if len(pod_posts) <= nposts:
```
Comment (timgrossmann) [inline instapy/instapy.py:5335]: Sorry, @ishandutta2007 could you please check the section below here if the changes are merged correctly.
I see some changes that might be "old".

**Comment by timgrossmann** [comment]
@ishandutta2007 Thank you very much, really appreciate it! 👍

**Comment by ishandutta2007** [comment]
Hey it need to be tested again, I am facing some db issue, not sure where from it crept in, this one https://github.com/timgrossmann/InstaPy/issues/4328

**Comment by timgrossmann** [comment]
Damn it, sorry.
I tested this before it needed rebasing it worked then.

However, it's "only" in dev, haven't merged it to master.

**Comment by ishandutta2007** [comment]
yes the dev branch has quite a few changes, don't merge it yet, they need to be tested together.

**Comment by timgrossmann** [comment]
@ishandutta2007 Absolutely!
Thank you

**Pull request merged**